README.md

```diff
@@ -1,2 +1,77 @@
-# To-Do-List
-"To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. 
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="./murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>To do list</b></h3>
+
+</div>
+
+# 📖 <a name="To-do-list"></a>
+
+> "To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## 🛠 Built With <a name="built-with">HTML, CSS, & Javascript</a>
+
+- It has implemented Js Dynamic function to take data
+
+### Key Features <a name="key-features"></a>
+
+- Clean and interactive
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+All you need VScompiler or a compiler of your choice.
+use command: npm run build
+
+### Clone
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/felixDev22/To-do-list
+
+### Run
+
+To run project, run the following command: git clone
+
+Open in desired editor.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] when following these steps the potential user is not able to run the project, please make the instructions clear on how to run the project locally.